settings.gradle.kts

```diff
@@ -1,6 +1,7 @@
 plugins {
     id("com.gradle.enterprise") version "3.12.3"
     id("com.gradle.common-custom-user-data-gradle-plugin") version "1.8.2"
+    id("org.gradle.toolchains.foojay-resolver-convention") version "0.4.0"
```

Comment:
Why is that required for the Gradle upgrade?

Reply:
I was also surprised, but auto-discovery was not working properly on jenkins: https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fgradle-plugin/detail/PR-243/6/pipeline/21